readme.md

```diff
@@ -1,7 +1,276 @@
-## An attempt to find an interoperable CSSinJS format.
+## Interoperable CSSinJS format.
 
-The biggest issue at scale we currently have with all the CSSinJS implementations is that they are not sharable without the runtime.
+The biggest issue at scale we currently have with all CSSinJS implementations is that they are not sharable without the runtime.
 
-If we could find a format that is fast an easy to parse and supports all needs of current CSSinJS implementations, it could become a de facto standard for exported CSSinJS.
+We can define a format that is easy to parse and supports all the needs of the current CSSinJS implementations on top of CSS.
 
-It could be similar to how CommonJS and ES3 are currently the standard for sharing JavaScript modules on npm.
+Package maintainers will be able to publish JavaScript files to NPM with this format inside instead of plain CSS.
+
+## First draft
+
+Format is optimized for parsing in JavaScript, not for human readability.
+
+### Markers
+
+Examples are using constant names instead of values for readability within the spec. The resulting format will use their values.
+
+```js
+const RULE_START = 0
+const RULE_END = 1
+const SELECTOR = 2
+const NAME = 3
+const PROPERTY = 4
+const VALUE = 5
+const PARENT_SELECTOR = 6
+const SPACE = 7
```

Comment:
As the combinators amount is limited to 5, I would probably go for separate constants, because this will make the result more compact.